app/templates/components/job-controls.hbs

```diff
@@ -6,16 +6,20 @@
     {{#if error}}
       {{#bs-alert type="danger"}}<span class="error-message">{{error}}</span>{{/bs-alert}}
     {{/if}}
-    {{#if job.isNew}}
+    {{#if job.isAuthorized}}
```

Comment:
This __if__ controls showing the  `Your job is ready. Click <strong>Start</strong> to submit. You will be charged at a rate of $x per hour of computation. Results will be uploaded to Duke Data Service.` message.

The following `{{#if job.isNew}}` shows the token prompt otherwise it shows the buttons.